### ISSUE TITLE
Support for yarn v2 pnp

### ISSUE THREAD

**Issue by paulober** [microsoft/vscode-extension-test-runner#52]
Currently with yarn v2 pnp you get an error in the test panel: `Error: Can't resolve '@vscode/test-cli' in ...`

**Comment by flying-sheep** [comment]
Yeah. How do I configure the `node` executable that runs the `.vscode-test.mjs`?
